### ISSUE TITLE
Inform user when analysis fails

### ISSUE THREAD

**Issue by ebugden** [theia-ide/theia-trace-extension#460]
Currently, if the analysis fails (i.e. treeResponse.status is FAILED) then the analysis's view panel never updates and shows "Analysis running" forever even though the server has responded. 

* **When the analysis status returned is FAILED, update the view panel to display a message (ex. "Analysis failed.")** 
* This needs to happen for all chart types (not just XY charts)
* If possible, add detail to the message to give the user some indication of why the analysis failed. Is the analysis broken? Was processing interrupted? The user's first reflex is to think that it's their fault the analysis failed (ex. Their trace files are the wrong format), so if it's not the user's fault this should be clearly communicated. 

https://user-images.githubusercontent.com/28311615/130871089-d9f0e8a3-05ac-40cb-a4ce-f2bf601a74ee.mp4

# To reproduce
This infinite spinning can be observed **the first time** the Event Matching Latency analysis is opened with the tracevizlab's apt trace.

* **Trace:** apt (from the tracevizlabs, can use this project's gitpod demo to open the apt trace)
* **Analysis:** Event Matching Latency

⚠️ **To re-observe, close the trace in the trace server** (right-click --> "Delete Trace" in the Opened Traces menu). **Event Matching Latency only returns status FAILED the first time** the analysis is opened on server (0:26 - 0:30), otherwise it returns the status COMPLETED (0:37 - 0:40).

https://user-images.githubusercontent.com/28311615/130796784-624f5c6d-48d8-4cb6-9a9b-4703d439c61f.mp4

# Implementation
This fix needs to be applied to all chart types (all output components). Ideally adding this message wouldn't require modifying each output component separately.
* PR #447 can be used as an example for which areas of the code to change. However this approach requires modifying each output component separately (code duplication --> more of a hassle to change) which isn't ideal. 
  * The CSS styling also seems to be messy in this PR (unnecessary contradiction/duplication with style tags) so that could be cleaned up as well.
   * Add the code in the [renderChart( ) function in the xy-output-component](https://github.com/theia-ide/theia-trace-extension/blob/f27170908b76bbbb1bfb6c37b3f84eb074a17886/packages/react-components/src/components/xy-output-component.tsx#L139).

## Pseudocode 
```typescript
renderChart {
   // ...
   if (this.state.outputStatus === ResponseStatus.FAILED) {
      // Show "analysis failed" message.
   }
   // ...
}
```

**Comment by jeffGodonou** [comment]
The issue of the infinite spinning doesn't happen when the code is cloned and run on a local machine. After a test with @IbrahimFradj , we came to the supposition that it is an issue appearing only on Gitpod.

https://user-images.githubusercontent.com/25375286/131527913-1111bd07-7f39-4c2f-8eff-f7b21cea4bd0.mp4

**Comment by bhufmann** [comment]
> The issue of the infinite spinning the first time the analysis is opened doesn't happen when the code is cloned and run on a local machine. After a test with @IbrahimFradj , we came to the supposition that it is an issue appearing only on Gitpod.
> Ubuntu.64-bit.-.VMware.Workstation.2021-08-31.11-02-48.mp4

I think it's a timing issue for the event matching latency in the trace server due to concurrent running threads, which makes it happen only sometimes. I have noticed this issue on my laptop as well. 

In the end, failures have to be handled better and that's what the issue is about. The event matching latency is just an example of a failing case.

**Comment by ebugden** [comment]
Resolved by PR #574. Forgot to link this issue in the PR!
